### PR TITLE
Add trailing comma after arguments

### PR DESCRIPTION
**Pull request by apalski**
PR to fix Rubobop violations for no consistent comma after arguments.
Feedback welcome.

https://trello.com/c/QcYj6OJ6
Thanks for taking a look!

### CODE REVIEW

**Comment by apalski** [comment]
Merging now, please add any comments to this PR and I will review them!

File: spec/features/admin/admin_deletes_user_spec.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
     expect(page).
       to have_text(I18n.t(
         "flash.actions.destroy.notice",
-        resource_name: "User"
-      ))
+        resource_name: "User",
+      ),)
```
Comment (elle) [inline spec/features/admin/admin_deletes_user_spec.rb]: Suggestion to change the format to be:
```
    notice = I18n.t("flash.actions.destroy.notice", resource_name: "User")
    expect(page).to have_text(notice)
```

File: spec/features/admin/admin_edits_user_spec.rb [hunk 2]
```diff
@@ -20,8 +20,8 @@
     expect(page).
       to have_text(I18n.t(
         "flash.actions.update.notice",
-        resource_name: "User"
-      ))
+        resource_name: "User",
+      ),)
```
Comment (elle) [inline spec/features/admin/admin_edits_user_spec.rb]: Same ^

File: spec/features/users/user_creates_new_account_spec.rb [hunk 3]
```diff
@@ -16,7 +16,7 @@
     expect(page).not_to have_text(I18n.t(
       "flash.actions.create.notice",
       resource_name: "User",
-    ))
+    ),)
```
Comment (elle) [inline spec/features/users/user_creates_new_account_spec.rb]: And ^

File: spec/features/users/user_updates_profile_spec.rb [hunk 4]
```diff
@@ -15,7 +15,7 @@
     expect(page).
       to have_text(I18n.t(
         "flash.actions.update.notice",
-        resource_name: "User"
-      ))
+        resource_name: "User",
+      ),)
```
Comment (elle) [inline spec/features/users/user_updates_profile_spec.rb]: Lastly ^

File: spec/features/admin/admin_deletes_user_spec.rb (hunk 1 above)
Comment (apalski) [reply spec/features/admin/admin_deletes_user_spec.rb]: Done in PR ad-refactor-flash-format-specs sha: 4be77c8

File: spec/features/admin/admin_edits_user_spec.rb (hunk 2 above)
Comment (apalski) [reply spec/features/admin/admin_edits_user_spec.rb]: Done in PR ad-refactor-flash-format-specs sha: 4be77c8

File: spec/features/users/user_creates_new_account_spec.rb (hunk 3 above)
Comment (apalski) [reply spec/features/users/user_creates_new_account_spec.rb]: Done in PR ad-refactor-flash-format-specs sha: 4be77c8

File: spec/features/users/user_updates_profile_spec.rb (hunk 4 above)
Comment (apalski) [reply spec/features/users/user_updates_profile_spec.rb]: Done in PR ad-refactor-flash-format-specs sha: 4be77c8

**Pull request merged**